Tests/FluentAssertions.Specs/Specialized/TaskCompletionSourceAssertionSpecs.cs

```diff
@@ -88,5 +88,51 @@ public async Task When_TCS_is_null_it_should_fail()
             await action.Should().ThrowAsync<XunitException>()
                 .WithMessage("Expected subject to complete within 1s, but found <null>.");
         }
+
+        [Fact]
+        public async Task When_TCS_completes_in_time_and_it_is_not_expected_it_should_fail()
+        {
+            // Arrange
+            var subject = new TaskCompletionSource<bool>();
+            var timer = new FakeClock();
+
+            // Act
+            Func<Task> action = () => subject.Should(timer).NotCompleteWithinAsync(1.Seconds());
+            subject.SetResult(true);
+            timer.Complete();
+
+            // Assert
+            await action.Should().ThrowAsync<XunitException>().WithMessage("*to not complete within*");
+        }
+
+        [Fact]
+        public async Task When_TCS_did_not_complete_in_time_and_it_is_not_expected_it_should_succeed()
+        {
+            // Arrange
+            var subject = new TaskCompletionSource<bool>();
+            var timer = new FakeClock();
+
+            // Act
+            Func<Task> action = () => subject.Should(timer).NotCompleteWithinAsync(1.Seconds(), "test {0}", "testArg");
```

Comment:
Should the `, "test {0}", "testArg"` be in the two other tests?

Reply:
yep